python/build-wheel-macos.sh

```diff
@@ -13,52 +13,57 @@ MACPYTHON_URL=https://www.python.org/ftp/python
 MACPYTHON_PY_PREFIX=/Library/Frameworks/Python.framework/Versions
 DOWNLOAD_DIR=python_downloads
 
-PY_VERSIONS=("2.7.13"
-             "3.4.4"
-             "3.5.3"
-             "3.6.1")
-PY_INSTS=("python-2.7.13-macosx10.6.pkg"
-          "python-3.4.4-macosx10.6.pkg"
-          "python-3.5.3-macosx10.6.pkg"
-          "python-3.6.1-macosx10.6.pkg")
-PY_MMS=("2.7"
-        "3.4"
-        "3.5"
-        "3.6")
+if [[ "$PYTHON" == "2.7" ]]; then
```

Comment:
It would be nice to leave the default behavior of this script unchanged so that it builds all of the wheels by default.

Instead of reading environment variables, one option would be to call this script from Travis and pass in different command line arguments.

Reply:
This script is intended to be useful outside of Travis, where we can't assume that the `$PYTHON` environment variables is set.

Reply:
Oh, I see! Thanks for context; I'll fix up sometime in the next day.

Reply:
Agree that environment variables could be problematic, so I implemented a simple command line argument. But I forgot that Travis has a nice column that tells you what environment variables are set for the build job; with the command line argument only, I see four jobs that all look (superficially) the same.

So what we could do is check if an environment variable is set; if yes, check that it's a valid version of Python and build the wheel for just that version. We could even choose something besides ```$PYTHON``` if that might be overloaded.

Up to you. I could definitely see the long-term use for using a CLI for build scripts. I'll let the cli version keep testing on Travis.

Reply:
Or derp, we could keep the CLI argument and pass in a Travis environment variable. That may be the best option.